StrideLearningApp/View Controllers/NewMessageController.swift

```diff
@@ -68,6 +72,8 @@ class NewMessageController: UITableViewController, UISearchResultsUpdating, UISe
         searchController.searchBar.sizeToFit()
         tableView.tableHeaderView = searchController.searchBar
     }
+    
+    
```

Comment:
and these two blank lines. better to have code condensed and have unnecessary white space